src/components/PostList/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from '../Post';
+import './PostList.css';
+
+function PostList({ posts }) {
+  return (
+    <ul className="postList">
+      {posts.map(post => <Post key={post.id} post={post} />)}
```

Comment:
```suggestion
      {posts.map(post => <Post key={post.id} post={...post} />)}
```
якщо так написати можна буде у компоненті Post робити деструтуризацію об'єкта post в пропсах